### DISCUSSION TITLE
Hosted Linkace

### DISCUSSION THREAD

**Discussion by npivic** [Kovah/LinkAce#612] [General]
Hi!

I know a hosted version of Linkace is in the works. I'm just curious: how far into the future might one be realised?

I'd love to pay to use a hosted Linkace rather than host my own.

Thanks a lot for making Linkace.

**Comment by Kovah** [comment]
Hi!
I have put the entire hosting idea on hold at the moment. There are various reasons for this. Biggest one is the rather small interest so far, in contrast to the huge amount of work that would be needed to bring this to life. Unfortunately, the "market" for bookmarking services is quite challenging too, with services like Raindrop offering cheap yearly plans for like 30€ bucks a year (= 2,5€ per month).

If you have no server at home, you could rent mini servers [like on Vultr](https://www.vultr.com/pricing/#cloud-compute/) for little money and install LinkAce there.

I would like to encourage interested users to upvote this and share their expectations and budgets.

**Reply by pivic** [reply]
Thanks! I've got my installation on vultr at the moment, just have to get the broken links feature to work.

I might write a guide on how to set up your own installation on vultr...
